identity_iota_core/packages/iota_identity/sources/identity.move

```diff
@@ -24,6 +24,24 @@ module iota_identity::identity {
     /// The controller list must contain at least 1 element.
     const EInvalidControllersList: u64 = 3;
 
+    // ===== Events ======
+    /// Event emitted when a `identity`'s `Proposal` with `ID` `proposal` is created or executed by `controller`.
+    public struct ProposalEvent has copy, drop {
+        identity: ID,
+        controller: ID,
+        proposal: ID,
+        // Set to `true` if `proposal` has been executed.
+        executed: bool,
+    }
+
+    /// Event emitted when a `Proposal` with `ID` `proposal` has reached the AC threshold and
+    /// can now be executed.
+    /// `T` is the proposal's action type.
```

Comment:
What `T`? ^^

Reply:
Ooof the event type had a phantom type parameter T that I later removed apparently without updating the inline docs. Thank you